src/puppetlabs/puppetdb/scf/storage.clj

```diff
@@ -1212,7 +1221,7 @@
                :order-by [[time-field :desc]]
                :limit 1}]
     (sql/with-query-results result-set (hcore/format query)
-      (time-field (first result-set)))))
+      (some-> result-set first time-field))))
```

Comment:
Also why `some->` instead of `->`?
Also also on a separate note, `query-to-vec` and `hcore/format` would be equivalent to `with-query-results` + `hcore/format` here, (another use case for the helper i mentioned in your other PR